compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Window.desktop.kt

```diff
@@ -638,7 +483,7 @@ fun Window(
 @Stable
 interface FrameWindowScope : WindowScope {
     /**
-     * [ComposeWindow] that was created inside [androidx.compose.ui.window.Window].
+     * [AwtWindow] that was created inside [androidx.compose.ui.window.Window].
```

Comment:
```suggestion
     * [ComposeWindow] that was created inside [androidx.compose.ui.window.Window].
```

`AwtWindow` is incorrect. Here we describe the class.

Reply:
Good catch. The refactoring changed it...